plugins/ksp/src/org/jetbrains/kotlin/ksp/symbol/impl/binary/KSFunctionDeclarationDescriptorImpl.kt

```diff
@@ -90,6 +90,10 @@ class KSFunctionDeclarationDescriptorImpl(val descriptor: FunctionDescriptor) :
         }
     }
 
+    override val isAbstract: Boolean by lazy {
+        this.modifiers.contains(Modifier.ABSTRACT)
```

Comment:
Does descriptor add the abstract modifier automatically for interfaces?

Reply:
Yes, I tested "contains" in test case for covering descriptor.

Reply:
List is a builtin and may have a different behavior than regular interfaces. Can you check with something like `kotlin.sequences.Sequence.iterator()`?